library/system/src/lib/yast2/system_service.rb

```diff
@@ -49,20 +49,15 @@ module Yast2
   class SystemService
     extend Forwardable
 
-    # @return [Yast::SystemdService]
+    # @return [Yast::SystemdServiceClass::Service]
     attr_reader :service
 
-    def_delegators :@service, :running?, :name, :description
+    # @return [Hash<Symbol,Object>] Errors when trying to write changes to the
+    #   underlying system.
+    attr_reader :errors
 
-    # @!method state
-    #
-    # @return [String]
-    def_delegator :@service, :active_state, :state
-
-    # @!method substate
-    #
-    # @return [String]
-    def_delegator :@service, :sub_state, :substate
```

Comment:
I think we should not remove those lines. Systemd methods `#active_state` and `#sub_state` are mapped to `#state` and `#substate`. The idea was to avoid the usage of systemd names but more general ones.

Reply:
In fact, there should be code in yast-services-manager using those methods.